### PR TITLE
#163388088 Set up an empty react app

### PR DESCRIPTION
**Pull request by dev-jey**
#### What does this PR do?
It creates an empty react app with linting rules
#### Description of Task to be completed?
- use create-react-app tool to initiate an empty react project
 - configure eslint
#### How should this be manually tested?
1. Clone the repo
```
git clone https://github.com/andela/ah-the-phoenix-frontend.git
```
2. Change it into the current directory
```
cd ah-the-phoenix-frontend
```
3. Checkout to the chore branch
```    
git checkout ch-create-react-app-163388088
```
4. Install the required dependencies using 
```
npm install
```
5. Run the local development server using 
```
npm start 
```
and you should get the following message:
```
Compiled successfully!

You can now view ah-the-phoenix-frontend in the browser.

  Local:            http://localhost:3000/
  On Your Network:  http://10.10.160.83:3000/

Note that the development build is not optimized.
To create a production build, use npm run build.
```

Note that the development build is not optimized.
To create a production build, use
```
 npm run build
```
Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
[#163388088](https://www.pivotaltracker.com/story/show/163388088)

Screenshots (if appropriate)
N/A

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    "semi": ["error", "always"],
```
Comment (ezrqnkemboi) [inline .eslintrc.js]: Let's add `"no-trailing-spaces": 0,` to make sure that we do not allow trailing spaces.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
```
Comment (ezrqnkemboi) [inline .eslintrc.js]: Add jest as part of env.

Comment (dev-jey) [reply .eslintrc.js:4]: working on it

Comment (ezrqnkemboi) [reply .eslintrc.js]: Push changes after it is done and let at least one person review the PR.

**Pull request merged**